content/people.md

```diff
@@ -50,7 +50,6 @@ id = "people"
           </div>
         </div>
       </div>   
-    </div>
```

Comment:
Why is this removed - @ciflikli should it be in or not? If it should stay, please revert this line.

Reply:
Yes, removing the ```</div>``` currently breaks the alignment. I am looking for a responsive bootstrap solution; in the meanwhile I will revert the line.